interoperability.md

```diff
@@ -0,0 +1,39 @@
+# Assess interoperability of Presentation API implementations
+
+This document discusses the interoperability of potential Presentation API implementations in particular interoperability
```

Comment:
This is a nit-pick, but wrapping the text with shorter lines will make future diffs easier to read.